footprint: [[0.1, 0.0], [0.0, 0.1], [0.0, -0.1], [-0.1, 0.0]]
robot_radius: 0.10

transform_tolerance: 0.5
update_frequency: 5.0
publish_frequency: 5.0

global_frame: map
robot_base_frame: base_link
resolution: 0.05

rolling_window: false
track_unknown_space: true

plugins:

    - {name: explore_boundary, type: "frontier_exploration::BoundedExploreLayer"}
    - {name: sensor,           type: "costmap_2d::ObstacleLayer"}
    - {name: inflation,        type: "costmap_2d::InflationLayer"}

explore_boundary:
    resize_to_boundary: true
    frontier_travel_point: closest

sensor:
    observation_sources: laser
    laser: {data_type: LaserScan, clearing: true, marking: true, topic: scan, inf_is_valid: true}

inflation:
    inflation_radius: 1.5
